cardano-cli/test/cardano-cli-golden/Test/Golden/ErrorsSpec.hs

```diff
@@ -90,30 +89,22 @@ test_EraBasedDelegationError :: TestTree
 test_EraBasedDelegationError =
   testErrorMessagesRendering "Cardano.CLI.EraBased.Run.Certificate" "EraBasedDelegationError"
     [ ("EraBasedDelegReadError"
-      , EraBasedDelegReadError $ FileError "path/file.txt" InputInvalidError)
-    , ("EraBasedCredentialError1"
-      , EraBasedCredentialError
-        $ ShelleyStakeAddressCmdReadKeyFileError $ FileError "path/file.txt" InputInvalidError)
-    , ("EraBasedCredentialError2"
-      , EraBasedCredentialError
-        $ ShelleyStakeAddressCmdReadScriptFileError
+      , EraBasedDelegReadError
+      $ FileError "path/file.txt" InputInvalidError)
+    , ("EraBasedDelegationStakeCredentialError1"
+      , EraBasedDelegationStakeCredentialError
+        $ StakeCredentialInputDecodeError
+        $ FileError "path/file.txt" InputInvalidError)
+    , ("EraBasedDelegationStakeCredentialError2"
+      , EraBasedDelegationStakeCredentialError
+        $ StakeCredentialScriptDecodeError
         $ FileError "path/file.txt"
         $ ScriptDecodeSimpleScriptError
         $ JsonDecodeError "json decode error")
-    , ("EraBasedCredentialError3"
-      , EraBasedCredentialError
-        $ ShelleyStakeAddressCmdReadKeyFileError $ FileError "path/file.txt" InputInvalidError)
-    , ("EraBasedCredentialError4"
-      , EraBasedCredentialError
-        $ ShelleyStakeAddressCmdWriteFileError $ FileError "path/file.txt" ())
-    , ("EraBasedCredentialError5"
-      , EraBasedCredentialError
-        $ StakeRegistrationError StakeAddressRegistrationDepositRequired)
-    , ("EraBasedCredentialError6"
-      , EraBasedCredentialError
-        $ StakeDelegationError
-        $ VoteDelegationNotSupported
-        $ AnyShelleyToBabbageEra ShelleyToBabbageEraShelley)
```

Comment:
Delete some not sensical golden tests.  For example delegation and registration are different things it's possible to construct a delegation error type that reports a registration error.